src/tests/getPersonaIdentifiers/getPersonaIdentifiers.test.ts

```diff
@@ -0,0 +1,120 @@
+// tslint:disable:no-magic-numbers
+// tslint:disable:max-file-line-count
+import * as assert from 'assert';
+import CreateIdentifierOptions from '../../serviceFactory/options/CreateIdentifierOptions';
+import setup from '../utils/setup';
+import {
+  TEST_ORGANISATION,
+  TEST_PERSONA_ID,
+} from '../utils/values';
+
+describe('getPersonaIdentifiers', () => {
+  const persona = TEST_PERSONA_ID;
+  const service = setup();
+  const identifierData1: CreateIdentifierOptions = {
+    ifi: {
+      key: 'mbox',
+      value: 'mbox:test1@mbox.com',
+    },
+    organisation: TEST_ORGANISATION,
+    persona,
+  };
+  const identifierData2: CreateIdentifierOptions = {
+    ifi: {
+      key: 'mbox',
+      value: 'mbox:test2@mbox.com',
+    },
+    organisation: TEST_ORGANISATION,
+    persona,
+  };
+  const identifierData3: CreateIdentifierOptions = {
+    ifi: {
+      key: 'mbox',
+      value: 'mbox:test3@mbox.com',
+    },
+    organisation: TEST_ORGANISATION,
+    persona,
+  };
+
+  beforeEach(async () => {
+    await service.clearService();
+  });
+
+  after(async () => {
+    await service.clearService();
+  });
+
+  it('should get all persona identifiers with default options', async () => {
+    const { identifier: identifier1 } = await service.createIdentifier(identifierData1);
+    const { identifier: identifier2 } = await service.createIdentifier(identifierData2);
+    await service.createIdentifier(identifierData3);
+
+    const result = await service.getPersonaIdentifiers({
+      organisation: TEST_ORGANISATION,
+      persona,
+    });
+
+    assert.equal(result.identifiers.length, 3); // tslint:disable-line:no-magic-numbers
```

Comment:
Why are we asserting a length of three but asserting two?

Reply:
We do this in other tests below too.

Reply:
I guess that if the first 2 are correct then there is only one left so it must be the third. Might as well assert it though, I agree.

Reply:
Assertions added

Reply:
Noice 👍